home/templates/includes/navigation.html

```diff
@@ -188,71 +188,55 @@
           <li class="nav-item">
             <a href="{% url 'about_us' %}" class="nav-link"> About </a>
           </li>
-          <li class="nav-item dropdown">
+          <li class="nav-item dropdown1">
             <a
               href="#"
               class="nav-link dropdown-toggle"
-              id="frontPagesDropdown"
+              id="projectsDropdown"
               aria-expanded="false"
-              data-bs-toggle="dropdown"
             >
               Projects
               <span class="fas fa-angle-down nav-link-arrow ms-1"></span>
             </a>
-            <div
-              class="dropdown-menu px-0 py-2 p-lg-4"
-              aria-labelledby="frontPagesDropdown"
-            >
+            <div class="dropdown-menu1 px-0 py-2 p-lg-4" aria-labelledby="projectsDropdown">

```

Comment:
Use "dropdown-menu" instead of "dropdown-menu1"

Reply:
the class with the name of "dropdown-menu" is a built in class which conflicts with my current functionality, so I used this class to differentiate from conflict.